src/lib/y2storage/fake_device_factory.rb

```diff
@@ -581,7 +571,7 @@ def create_lvm_lv_stripe_parameters(lvm_lv, args)
       lvm_lv.stripes = stripes if stripes > 0
 
       stripe_size = args["stripe_size"] || DiskSize.zero
-      lvm_lv.stripe_size = stripe_size.size if stripe_size.size > 0
+      lvm_lv.stripe_size = stripe_size if stripe_size.to_i > 0
```

Comment:
The code as it is now will work in both cases - either if stripe_size is an integer (bytes) or if is a DiskSize object. So I would rather leave it as it is.